ftva_lab_data/views.py

```diff
@@ -432,3 +433,57 @@ def get_alma_data(request: HttpRequest, inventory_number: str) -> list[dict]:
     # TODO: return a template with the records
     # For now, just return the list of dictionaries
     return full_data_dicts
+
+
+def get_filemaker_data(request: HttpRequest, inventory_number: str) -> list[dict]:
+    """Fetch records using FilemakerClient.
+
+    :param request: The HTTP request object.
+    :param inventory_number: The inventory number to search for in Filemaker.
+    :return: a list of dictionaries containing Filemaker record data, each with keys
+        "record_id", "title", and "full_data".
+    """
+
+    user = settings.FILEMAKER_USER
+    password = settings.FILEMAKER_PASSWORD
+    fm_client = FilemakerClient(user=user, password=password)
+
+    records = fm_client.search_by_inventory_number(inventory_number)
+
+    # List of required fields, as defined by FTVA
+    filemaker_fields = [
+        "type",
+        "inventory_no",
+        "inventory_id",
+        "format_type",
+        "title",
+        "aka",
+        "director",
+        "episode_title",
+        "production_type",
+        "Acquisition type",
+        "Alma",
+        "availability",
+        "release_broadcast_year",
+        "notes",
+        "element_info",
+        "spac",
+        "episode no.",
+        "film base",
+        "donor_code",
+    ]
+    full_data_dicts = []
+
+    for record in records:
+        record = get_specific_filemaker_fields(record, filemaker_fields)
```

Comment:
I'd like it to be clearer that this is a set of filtered fields, especially as the code in the next few lines is acting on this reduced record, not the original one in the `for record in records` loop.  The fact that `record` changes from a (Filemker) `Record` on line 477 to a `dict` on line 478 adds potential for confusion.  Since the method is `get_specific_filemaker_fields`, rename the variable to `filemaker_fields` or `fm_fields` to help make this clearer.